gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+networkTimeout=10000
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
in other nessie repos we seem to:

a) also commit `wrapper.jar` https://github.com/projectnessie/nessie/tree/main/gradle/wrapper

b) also set `distributionSha256Sum` https://github.com/projectnessie/nessie/blob/main/gradle/wrapper/gradle-wrapper.properties

Reply:
done. Good observation.